avail-subxt/examples/submit_data.rs

```diff
@@ -0,0 +1,51 @@
+use sp_keyring::AccountKeyring;
+use subxt::{tx::PairSigner, OnlineClient};
+
+use avail_subxt::*;
+
+use avail_subxt::avail::runtime_types::frame_support::storage::bounded_vec::BoundedVec;
+
+/// This example gets all the headers from testnet. It requests them in concurrently in batches of BATCH_NUM.
```

Comment:
The comment might be stale. Can we remove it and add some comment about what this script does?

Reply:
done